image/upload/internal/cmd/measurementsenvelope.go

```diff
@@ -96,6 +102,94 @@ func runEnvelopeMeasurements(cmd *cobra.Command, _ []string) error {
 	return nil
 }
 
+func overrideStaticMeasurements(in measurements.M, csp cloudprovider.Provider, attestationVariant string) (measurements.M, error) {
```

Comment:
We might want to define the PCRs of this function as const/vars somewhere instead for easier management

Reply:
Moved this as vars in `internal/attestation/measurements/overrides.go`